boards/mc-pic32mx-lccbuffer/ISRwrapper.inc

```diff
@@ -1 +0,0 @@
-../microchip-pic32mx-generic/ISRwrapper.inc
```

Comment:
I generally prefer to keep the link from boards/xxx-lccbuffer to boards/xxx-generic, because that means xxx-lccbuffer shows me all the files that I need to add to the target directory in a new application. No need to guess which files I should be taking from where.